sdk/keyvault/azure-keyvault-certificates/src/main/java/com/azure/security/keyvault/certificates/implementation/AzureKeyVaultConfiguration.java

```diff
@@ -0,0 +1,10 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.security.keyvault.certificates.implementation;
```

Comment:
This class is also in ` com.azure.security.keyvault.certificates` package. One of them can be deleted.